iphone/Classes/TiUILabel.m

```diff
@@ -436,9 +436,8 @@ -(void)setColor_:(id)color
 -(void)setEllipsize_:(id)value
 {
 	ENSURE_SINGLE_ARG(value, NSNumber);
-	//for bool case and parity with android
 	if ([TiUtils intValue:value] == 1) {
-		[[self label] setLineBreakMode:NSLineBreakByTruncatingTail];
+		[[self label] setLineBreakMode:NSLineBreakByCharWrapping];
```

Comment:
Instead of this, please check if the input is a bool and leave the if-block as it is, if so.